ui/StatusQ/src/typesregistration.cpp

```diff
@@ -113,6 +113,14 @@ void registerStatusQTypes() {
                                                   return new OnboardingEnums;
                                               });
 
+// create import alias from QtGraphicalEffects 1.15 to Qt5Compat.GraphicalEffects
+// in order to satisfy both Qt 5/6, relying the same version of qml code
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+    qmlRegisterModule("QtGraphicalEffects", 1, 15);
+    qmlRegisterModuleImport("QtGraphicalEffects", QQmlModuleImportModuleAny,
```

Comment:
I'd be explicit with `1` here for the major version, that's what we want to replace

Reply:
It's actually not replacing but attaching additional, implicit import to another. In that case w are attaching to the one we defined in one line above. So there is no need to repeat the number there imo.